app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -23,10 +23,11 @@ def call
       def access_token(input)
         request_body = AuthnOidc::AuthenticateRequestBody.new(input.request)
 
-        oidc_client_configuration = @get_oidc_client_configuration.(
-          redirect_uri: nil, # not needed for this request
-            service_id: input.service_id,
-            conjur_account: input.account
+        required_variable_names = %w(provider-uri id-token-user-property)
+        oidc_secrets            = @fetch_oidc_secrets.(
```

Comment:
I would remove the space between `oidc_secrets` and `=` here. This kind of indentation can be useful when you have lists of one-line variables being set but I don't think it's very useful here.

Reply:
done